index.html

```diff
@@ -9,7 +9,12 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
-  <h1></h1>
+  <div class="tech-container">
+    <div class="tech-container-content">
```

Comment:
delete this div, leave the  content inside of the `tech-container` div. also add the image by using an image tag here